tests/tools/network/backup/output.stdout

```diff
@@ -0,0 +1,89 @@
+[1m[36mBackup configuration of devices*************************************************
+[1m[34m* spine00.cmh ** changed : True ************************************************
+[1m[32mvvvv Backup configuration of devices ** changed : False vvvvvvvvvvvvvvvvvvvvvvvv INFO
+[1m[33m---- write_file ** changed : True ---------------------------------------------- INFO
+--- /tmp/spine00.cmh
+
```

Comment:
I see a potential issue with comparing the output against this file. A small change on the Brigade side will cause it to change and can be problematic with many tools and later different versions of Brigade. I.e. if we want to test against multiple versions.

Reply:
Yeah, let's fix when that happens. But I agree. Just don't to prematurely optimize this.